src/ontology/mondo-ingest-odk.yaml

```diff
@@ -36,7 +36,7 @@ components:
     - filename: ncit.owl
       source: http://purl.obolibrary.org/obo/ncit.owl
     - filename: omim.owl
-      source: https://github.com/monarch-initiative/omim/releases/latest/download/omim.ttl
+      source: https://github.com/monarch-initiative/omim/releases/latest/download/omim.owl
```

Comment:
## The change
The only change other than the ones from `make update_repo`. See: [commit](https://github.com/monarch-initiative/mondo-ingest/pull/433/commits/7ec335f0cdf927b0ccf656e11b68a530247ddad4)